containerm/util/containers_compare_test.go

```diff
@@ -423,150 +389,122 @@ func TestIsEqualHostConfig1(t *testing.T) {
 		},
 		"test_resources_equal": {
-			current: &types.Resources{
-				Memory:            "4m",
-				MemoryReservation: "3m",
-				MemorySwap:        "-1",
-			},
-			desired: &types.Resources{
-				Memory:            "4m",
-				MemoryReservation: "3m",
-				MemorySwap:        "-1",
-			},
+			current:        defaultResources,
+			desired:        defaultResources,
 			expectedResult: true,
 		},
 		"test_resources_not_equal": {
-			current: &types.Resources{
-				Memory:            "4m",
-				MemoryReservation: "3m",
-				MemorySwap:        "-1",
-			},
+			current: defaultResources,
 			desired: &types.Resources{
 				Memory:            "4m",
 				MemoryReservation: "3m",
 			},
-			expectedResult: false,
 		},
 	}
 
 	for testName, testCase := range testCases {
 		t.Run(testName, func(t *testing.T) {
-			res := isEqualResources(testCase.current, testCase.desired)
-			testutil.AssertEqual(t, testCase.expectedResult, res)
+			testutil.AssertEqual(t, testCase.expectedResult, isEqualResources(testCase.current, testCase.desired))
 		})
 	}
 }
 
 func TestIsEqualRestartPolicy(t *testing.T) {
+	defaultResPolicy := &types.RestartPolicy{
+		MaximumRetryCount: 0,
+		RetryTimeout:      0,
+		Type:              "always",
+	}
 	testCases := map[string]struct {
 		current        *types.RestartPolicy
 		desired        *types.RestartPolicy
 		expectedResult bool
 	}{
 		"test_current_nil_desired_nil": {
-			current:        nil,
-			desired:        nil,
 			expectedResult: true,
 		},
 		"test_current_nil_desired_not_nil": {
-			current:        nil,
-			desired:        &types.RestartPolicy{},
-			expectedResult: false,
+			desired: &types.RestartPolicy{},
 		},
 		"test_current_not_nil_desired_nil": {
-			current:        &types.RestartPolicy{},
-			desired:        nil,
-			expectedResult: false,
+			current: &types.RestartPolicy{},
 		},
 		"test_restartpolicy_equal": {
-			current: &types.RestartPolicy{
-				MaximumRetryCount: 0,
-				RetryTimeout:      0,
-				Type:              "always",
-			},
-
-			desired: &types.RestartPolicy{
-				MaximumRetryCount: 0,
-				RetryTimeout:      0,
-				Type:              "always",
-			},
+			current:        defaultResPolicy,
+			desired:        defaultResPolicy,
 			expectedResult: true,
 		},
 		"test_restartpolicy_not_equal": {
-			current: &types.RestartPolicy{
-				RetryTimeout: 0,
-				Type:         "always",
-			},
+			current: defaultResPolicy,
 			desired: &types.RestartPolicy{
 				RetryTimeout: 5,
 				Type:         "always",
 			},
-			expectedResult: false,
 		},
 	}
 
 	for testName, testCase := range testCases {
 		t.Run(testName, func(t *testing.T) {
-			res := isEqualRestartPolicy(testCase.current, testCase.desired)
-			testutil.AssertEqual(t, testCase.expectedResult, res)
+			testutil.AssertEqual(t, testCase.expectedResult, isEqualRestartPolicy(testCase.current, testCase.desired))
 		})
 	}
 }
 
 func TestIsEqualLog(t *testing.T) {
+	defaultDriverConf := &types.LogDriverConfiguration{
```

Comment:
```suggestion
	defaultDriverConfig := &types.LogDriverConfiguration{
```
So it can be aligned with `DriverConfig`, `ModeConfig`, etc.